model/clusters_mgmt/v1/autoscaler_resource_range_type.model

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct ResourceRange {
```

Comment:
I think the file should be named resource_range_type.model, also to indicate this is a generic resource that can be used outside of the autoscaler